src/test/java/uk/gov/hmcts/reform/unspec/bpmn/BpmnBaseTest.java

```diff
@@ -83,6 +84,19 @@ static void shutDown() {
         return engine.getExternalTaskService().createExternalTaskQuery().list();
     }
 
+    /**
+     * Retrieves a process definition which has a start message event with the messageName.
+     *
+     * @param messageName the name of the message.
+     * @return process definitions with given message start message event.
+     */
+    public ProcessDefinition getProcessDefinitionByMessage(String messageName) {
```

Comment:
worth enumerating messageName?

Reply:
We could use CaseEvents here, which most likely won't change, but I'm not sure if we want to have that dependency. I'd leave it as string, the worst thing that can happen is that definition won't be found.